cpp/cmake/thirdparty/CUSPATIAL_GetCUDF.cmake

```diff
@@ -0,0 +1,41 @@
+#=============================================================================
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+
+function(find_and_configure_cudf VERSION)
+    CPMFindPackage(NAME cudf
+        VERSION         ${VERSION}
+        GIT_REPOSITORY  https://github.com/rapidsai/cudf.git
+        GIT_TAG         branch-${VERSION}
+        GIT_SHALLOW     TRUE
+        SOURCE_SUBDIR   cpp
+        OPTIONS         "BUILD_TESTS ${BUILD_TESTS}"
```

Comment:
Do we actually want this to propagate or do we always want to set this to "NO"?

Reply:
We have to pass this right now because cudf only exports the `cudf::cudftestutil` target with this flag on.

Reply:
Sounds like something we should fix upstream in libcudf, but this works as a workaround for now.

Reply:
Fixed.